src/backend/base/langflow/langflow_launcher.py

```
@@ -0,0 +1,69 @@
+        if process and process.poll() is None:
+            try:
+                process.send_signal(signum)
+            except ProcessLookupError:
+                # Process already terminated
+                pass
+    
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+    
+    try:
+        process = subprocess.Popen(command, env=env)
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)